ee/codegen/src/generators/workflow.ts

```diff
@@ -758,6 +774,28 @@ export class Workflow {
             ],
           };
         }
+      } else if (graphMutableAst.type === "right_shift") {
+        if (graphMutableAst.lhs.type == "port_reference") {
```

Comment:
I definitely anticipate fancy recursion in a future PR, but am trying to build up to it by being explicit with test cases before doing it all at once